themes/bootstrap3/templates/admin/home.phtml

```diff
@@ -38,7 +38,8 @@
         <tr>
           <th><?=$this->transEsc('Last Modified')?>: </th>
           <?php $lastModified = $core->xpath('//lst[@name="' . $coreName . '"]/lst/date[@name="lastModified"]') ?>
-          <td><?=$this->escapeHtml($this->dateTime()->convertToDisplayDateAndTime(DATE_RFC3339_EXTENDED, (string)array_pop($lastModified)))?></td>
+          <td><?php //=$this->escapeHtml($this->dateTime()->convertToDisplayDateAndTime(DATE_RFC3339_EXTENDED, (string)array_pop($lastModified)))?></td>
```

Comment:
Is this meant to be commented out? Is this a fix to an unrelated problem that we should merge separately?

Reply:
This is unrelated to this PR, so I removed it. It was temporary fix, because my index didn't reported last modified date, so admin home page crashed. I don't know if it should be fixed. But probably could be fixed easily by adding a check to date existence.

Reply:
I added fix for such a situation here: https://github.com/vufind-org/vufind/pull/2485